lib/documents.js

```diff
@@ -290,12 +292,21 @@ export async function _generateInternalId() {
  *
```

Comment:
Please add a sentence to the docs for this function above after `Retrieves the encrypted registration...`. It should say "This function may be extremely slow to execute, so do not call it in any hot code paths."